src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProfileSteps.java

```diff
@@ -94,14 +105,14 @@ public void verifyMyUserNameIsUpdatedInTheTopMenu() {
     public void verifyUserInformationIsUpdatedInUserDropdownMenu() {
         Map<String, String> dropdownMenuInfo = profilePage.getTopMenu().getUserMenu().getUserInformationAsMap();
         SoftAssert softAssert = new SoftAssert();
-        softAssert.assertEquals("Underlying Initials of User Menu does not match with the Initials edited previously.",
-                dropdownMenuInfo.get("Underlying initials"), user.getInitials());
-        softAssert.assertEquals("The Username from User Menu does not match with the Username edited previously.",
-                dropdownMenuInfo.get("Details user name"), user.getUserName());
-        softAssert.assertEquals("The Name from User Menu does not match with the Name edited previously.",
-                dropdownMenuInfo.get("Details name"), user.getName());
-        softAssert.assertEquals("The Initials from User Menu does not match with the Initials edited previously.",
-                dropdownMenuInfo.get("Details initials"), user.getInitials());
+        softAssert.assertEquals(dropdownMenuInfo.get("Underlying initials"), user.getInitials(),
+                "Underlying Initials of User Menu does not match with the Initials edited previously.");
+        softAssert.assertEquals(dropdownMenuInfo.get("Details user name"), user.getUserName(),
+                "The Username from User Menu does not match with the Username edited previously.");
+        softAssert.assertEquals(dropdownMenuInfo.get("Details name"), user.getName(),
+                "The Name from User Menu does not match with the Name edited previously.");
+        softAssert.assertEquals(dropdownMenuInfo.get("Details initials"), user.getInitials(),
+                "The Initials from User Menu does not match with the Initials edited previously.");
         softAssert.assertAll();
     }
```

Comment:
Please add the getter method in the entity so, you can valiate all the information in the profile page